layouts/container.html

```diff
@@ -0,0 +1,22 @@
+<div class="container">
+  <div class="row">
+    <div class="col-md-2"></div>
+    <div class="col-md-8">
+      <div class="content">
+        <%= yield %>
+      </div>
+
+      <hr>
+
+      <footer>
+        <p class="pull-left">
+          &copy; Prometheus Authors 2016
```

Comment:
Ok.

Side note: for source code files, I recently read the year should describe the creation of a file, not the last modification.